### DISCUSSION TITLE
Welcome to ThinkPad-X1C8-Hackintosh Discussions!

### DISCUSSION THREAD

**Discussion by HJebbour** [HJebbour/ThinkPad-X1C8-Hackintosh#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jeofo** [comment]
I'm about to try and set my device up. I'll look for solutions for the internal mic, will update if I have anything. Thats the last piece of the puzzle, hopefully there is a way around it. Can you tell me some information you have on the mic so that I don't have to start from zero?

**Reply by HJebbour** [reply]
Hi, thank you for your interest in hackintoshing the X1C8. There's not much info about internal mic issue, but what I know is, the microphone is detected in system preferences but it doesn't work (like the mic is muted or something), and I heard somewhere that the mic connector (hw) is different from what it used to be and that is the reason why it doesn't work on macOS. I hope you find something about this issue and let me know if you need assistance.

**Reply by mccding** [reply]
我的 X1C8（20U9）无法使用 EFI。在 [EB|#LOG:EXITBS:START] 上失败。

**Comment by KSL0ng** [comment]
The EFI can not work for my X1C8（20UA).  failed on [EB|#LOG:EXITBS:START].
Can you tell me the version of your BIOS?

**Reply by HJebbour** [reply]
N2WET30W (1.20)

**Comment by Notxam77** [comment]
Working very well for me! Copied your EFI and generated my SMBIOS, core i5 model and using a samusng 980 pro SSD.

I tested sidecar and thunderbolt 3:

Sidecar does not work at all, the ipad shows up only when plugged in, not wirelessly, and still fails to connect. 

Thunderbolt tries to work, my dock lights up green, and my monitor flickers, but the computer display went dark and the USB devices were not connected.

**Comment by 7h1nKp4dx1Cg8** [comment]
The man is great at creating an OPEN CORE booster MAGIC,
![Screen Shot 2024-02-13 at 6 26 20 AM](https://github.com/HJebbour/ThinkPad-X1C8-Hackintosh/assets/159923755/fc832a1a-6d23-4233-9298-582d6a841bc1)
with the ThinkPad! This laptop may very well run MacOs quite better than the way it runs Windows 11! Almost as though built for macOS in the first place! Albeit the case this happens only after you apply the open core folder of HJebbour and only after such an applicitaceous advancement in ThinkPadology could one truly reap the benefit of this wonderful laptop and all the functionality of the macOS system integrated into one's personal computing experience! I know some people feel like this is a bit of a stretch. Still, I honestly CAN say, that I feel the macOS system (which I have installed on a separate partition from my Windows installation, obviously) could be faster and better than the Windows partition and could end up being the only OS on the laptop since I have gotten all of the functionality from this wonderful OC folder of HJebbour and now could potentially go without having a main windows partition and settle for only when I HAVE to use Windows for something, using a VM through either virtual box or Fusion! This has made my day this day and for that, I will tell you Sir HJebbour, thank you, thank you a million times, THANK YOU! Reluctant to say, id owned a standard "MacBook with" a release date of 2018 or so, and the damn thing broke on me and I've been without a MacOs laptop for some time now and so I truly appreciate your work, my friend! You are awesome dude!

**Comment by Crazy1wh** [comment]
Layout ID: Using 66 speaker will produce bass, which is much better than 71, but the microphone still doesn't work
